### ISSUE TITLE
E-mail auto sending.

### ISSUE THREAD

**Issue by Dovbo** [autogestion/corruption_tracker#11]
Organize e-mail letters auto sending to responsible goverment or municipal institution after verified claim incoming. These e-mails will be send to certain departments we can encrypt in polygons' ID in GEOjson.

**Comment by autogestion** [comment]
We can get some problems, if malefactors will use corrutpion tracker to spam goverment e-mails

Maybe better create something like report with pack of claims and send it once a day, for example

**Comment by Dovbo** [comment]
@autogestion Yep, it wil be correct. It looks exactly like "after verified claim incoming". Or rarer if we'll get info about  responsible person or department schedule.

**Comment by Vegasq** [comment]
> These e-mails will be send to certain departments we can encrypt in polygons' ID in GEOjson.

I'm not sure that it's good idea to keep such information in json file. More logical for me is to keep it id DB with direct mapping to geojson objects.

Also this way (when organization metadata is document in db) we will be able to provide UI for customers to fullfill information about organizations/add new organizations.

Don't know about original source of this geo json, but we also can create import/export tool to keep all this information in database.

**Comment by Dovbo** [comment]
Of course, e-mails will store in DB. We can group institutions with the same e-mail address for claims using field 'ID' in GEOjson. For example, ID=xxx001 means department A e-mail, ID=xxx002 - department B e-mail etc. All the claims added to polygons with IDs xxx001 once a day(week, month) will be sent by e-mail to department A, with IDs xxx002 to dept B. If we'll find not 1000 but less than 100 e-mails, ID format will be xxxx02, xxxx01 etc.
